src/decisionengine/framework/engine/ChannelWorkers.py

```diff
@@ -95,10 +92,10 @@ def setup_logger(self):
 
     def run(self):
         self.setup_logger()
-        self.task_manager.run()
+        self.task_manager.run_cycles()
```

Comment:
Should we emit a debug log message saying "this worker is starting tasks for XXX" here?